schemas/json/block.json

```
@@ -477,6 +477,10 @@
 					}
 				}
 			]
+		},
+		"styledClasses": {
+			"description": "An array of class names that have individual stylesheets. If defined, overrides the 'style' property on the frontend, loading styles only when the class is present.",
+			"type": "array"
```

Is it correct that the array contents are only strings?

If so, we can define the contents like this:

```suggestion
			"type": "array",
			"items": {
				"type": "string"
			}
```

---

Good point, thanks!